pkg/build/api/validation/validation_test.go

```diff
@@ -1762,7 +1872,7 @@ func TestValidateCommonSpec(t *testing.T) {
 				},
 			},
 		},
-		// 26
+		// 31
```

Comment:
Personally, I've started changing the test to use maps with human readable descriptions instead of indexes :wink: